zerver/tests/test_message_move_stream.py

```diff
@@ -1102,7 +1102,7 @@ def test_move_message_to_stream_and_topic(self) -> None:
             "iago", "test move stream", "new stream", "test"
         )
 
-        with self.assert_database_query_count(51), self.assert_memcached_count(14):
+        with self.assert_database_query_count(53), self.assert_memcached_count(14):
```

Comment:
This might be most appropriate as a follow-up commit to manage complexity, but I think we have the message IDs required to avoid having to fetch `ChannelTopic` rows for these automated messages. Does that theory sound right to you? If so, we can open a follow-up issue to track it.

Reply:
ya, I guess it makes sense